easy_toolbox.py

```diff
@@ -28,20 +28,20 @@
     ("build", "Build whole OIOIOI from source.", "build", True),
     ("up", "Run all SIO2 containers", "up -d"),
     ("down", "Stop all SIO2 containers", "down", True),
-    ("run", "Run server", "exec web python3 manage.py runserver 0.0.0.0:8000"),
-    ("bash", "Open command prompt on web container.", "exec web bash"),
-    ("bash_db", "Open command prompt on database container.", "exec db bash"),
+    ("run", "Run server", "exec{} web python3 manage.py runserver 0.0.0.0:8000"),
```

Comment:
this is quite cryptic, maybe it might be easier to use named params and to have {exec} param:
```"{exec} web python3 manage.py runserver 0.0.0.0:8000"```

and `{exec}` = `"exec"` or `"exec -T"` depending on the context